Testscripts/Linux/gpu-driver-install.sh

```diff
@@ -208,7 +208,7 @@ if [ $? -ne 0 ]; then
     exit 0
 fi
 
-if [[ $DISTRO == "redhat_8" ]]; then
+if [ -f /usr/libexec/platform-python ]; then
```

Comment:
This points to python3. I thought we uses python2. Please clarify.

Reply:
both versions work